command/login_test.go

```diff
@@ -435,16 +438,60 @@ func TestLoginCommand_Run(t *testing.T) {
 		client, closer := testVaultServer(t)
 		defer closer()
 
+		userClient, entityID, methodID, methodName := testhelpers.SetupLoginMFATOTP(t, client)
```

Comment:
BTW I just noticed that this is a 500 line test function, even before this PR.  There's actually no reason why it should be structured as a single test func with subtests, I'm not sure why it was done that way.  Consider breaking it up into individual funcs; if nothing else we'll save one level of indentation.